stdlib/printexc.mli

```diff
@@ -190,6 +190,22 @@ external get_callstack: int -> raw_backtrace = "caml_get_current_callstack"
     @since 4.01.0
 *)
 
+external get_kcallstack_deep :
```

Comment:
Are we sure we want these functions in `Printexc` or could they go into `Effect_handlers` in obj?
Just wondering if everything should be in obj while the API has a 'experimental' flavour. 

I might be missing a reason why this one needs to be here.

Reply:
The function returns a `raw_backtrace` which is only defined in `Printexc`, but `Obj` comes before `Printexc` in the dependency chain (see `stdlib/StdlibModules`). Hence I've left the functions in `Printexc`.

Reply:
ah, I see. Fair enough